homeassistant/components/dunehd/media_player.py

```diff
@@ -96,6 +112,16 @@ def available(self):
         """Return True if entity is available."""
         return bool(self._state)
 
+    @property
+    def unique_id(self):
+        """Return a unique_id for this entity."""
+        return self._unique_id
+
+    @property
+    def device_info(self):
+        """Return the device info."""
+        return self._device_info
```

Comment:
The device info could be here directly instead of `async_setup_entry` since you already have all the data needed in the class

Reply:
Fixed